app/Http/Controllers/VoteController.php

```
@@ -0,0 +1,66 @@
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(StoreVoteRequest $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)